System design refactoring (#55)
spectrum-finance/spectrum: docs/whitepaper/src/appendix.tex

> Validators in the Spectrum network consist of several committees, individually selected via the lottery mechanism.

Reviewer: ```suggestion
Validators in the Spectrum network fall into several committees, individually selected via the lottery mechanism.
```
